profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/binder/BaseProfitBricksRequestBinder.java

```diff
@@ -16,20 +16,18 @@
  */
 package org.apache.jclouds.profitbricks.rest.binder;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
 import java.util.Map;
-
 import org.jclouds.http.HttpRequest;
 import org.jclouds.rest.MapBinder;
-
 import com.google.common.base.Strings;
-
 import org.jclouds.io.MutableContentMetadata;
 import org.jclouds.io.payloads.BaseMutableContentMetadata;
+import static com.google.common.base.Preconditions.checkNotNull;
 
 public abstract class BaseProfitBricksRequestBinder<T> implements MapBinder {
 
+   protected final String endpointPath = "/rest/v2";
```

Comment:
Hmm weird. That provider is bound by default in the [LocationModule](https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/location/config/LocationModule.java#L85-L92) which is included by default in the [RestModule](https://github.com/jclouds/jclouds/blob/master/core/src/main/java/org/jclouds/rest/config/RestModule.java#L51) (the ruperclass of the `HttpApiModule`), so you should already have it.
Are you experiencing this issue when running the tests? If so, are you using a custom (small) Injector or a custom module set? If that is the case, try including the `LocationModule` in the lsit of modules used to create that small injector.